.gitmodules

```diff
@@ -17,6 +17,6 @@
 [submodule "mod/homepage/vendor/wodry"]
 	path = mod/homepage/vendor/wodry
 	url = https://github.com/srivig/wodry.git
-[submodule "mod/homepage/vendor/CountUp.js"]
-	path = mod/homepage/vendor/CountUp.js
-	url = https://github.com/inorganik/CountUp.js
+[submodule "mod/homepage/vendor/CountUp"]
+	path = mod/homepage/vendor/CountUp
+	url = https://github.com/inorganik/countUp.js.git
```

Comment:
we control the name of the submodule and the path (and we want them to be lowercase).  we _don't_ control their url.